.gitignore

```diff
@@ -1,8 +1,379 @@
-.idea/
-.vscode/
-*.iml
+# Created by https://www.toptal.com/developers/gitignore/api/pycharm,python,django
+# Edit at https://www.toptal.com/developers/gitignore?templates=pycharm,python,django
+
+### Django ###
+*.log
+*.pot
+*.pyc
+__pycache__/
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+media
+.dockerignore
```

Comment:
Including `.dockerignore` in `.gitignore` is not recommended. The `.dockerignore` file should be managed separately to ensure that unnecessary files are not included in Docker images. Please remove this line to meet the task requirements.